examples/model_checking.py

```diff
@@ -62,19 +62,23 @@ def get_simple_path(system, k):
     each time encodes a different state
     """
     res = []
-    for i in xrange(k):
+    for i in xrange(k+1):
         subs_i = get_subs(system, i)
-        for j in xrange(i+1, k):
+        for j in xrange(i+1, k+1):
+            state = []
             subs_j = get_subs(system, j)
             for v in system.variables:
                 v_i = v.substitute(subs_i)
                 v_j = v.substitute(subs_j)
-                res.append(Not(Equals(v_i, v_j)))
+                cond = Not(Iff(v_i, v_j)) if v_i.get_type() == BOOL else \
```

Comment:
You can use the function `EqualsOrIff` (see https://github.com/pysmt/pysmt/blob/master/pysmt/shortcuts.py#L361):
```python
Not(EqualsOrIff(v_i, v_j)) 
```